src/shippingservice/shippingservice_test.go

```diff
@@ -50,7 +50,7 @@ func TestGetQuote(t *testing.T) {
 	if err != nil {
 		t.Errorf("TestGetQuote (%v) failed", err)
 	}
-	if res.CostUsd.GetUnits() != 11 || res.CostUsd.GetNanos() != 220000000 {
+	if res.CostUsd.GetUnits() != 8 || res.CostUsd.GetNanos() != 990000000 {
 		t.Errorf("TestGetQuote: Quote value '%d.%d' does not match expected '%s'", res.CostUsd.GetUnits(), res.CostUsd.GetNanos(), "11.220000000")
```

Comment:
Should the expected be changed to 8.99 too?

Reply:
i changed it in my second commit!